src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -425,6 +425,18 @@
     <value>Actions</value>
     <comment>Header for the "actions" menu item. This navigates to a page that lets you see and modify commands, key bindings, and actions that can be done in the app.</comment>
   </data>
+  <data name="PreviewConnection_DefaultText" xml:space="preserve">
+    <value>Default text</value>
+  </data>
+  <data name="PreviewConnection_RedText" xml:space="preserve">
+    <value>Red text</value>
+  </data>
+  <data name="PreviewConnection_GreenText" xml:space="preserve">
+    <value>Green text</value>
+  </data>
+  <data name="PreviewConnection_BlueText" xml:space="preserve">
+    <value>Blue text</value>

```

Comment:
These might need explanatory comments

Reply:
Removed it since we no longer want localizable text for this